test/bigtest/network/config.js

```diff
@@ -333,16 +266,66 @@ export default function config() {
     ]
   });
 
+  this.get('/patron-notice-policy-storage/patron-notice-policies', {
```

Comment:
@OleksandrAntonenko1 could you please create a factory for this. The default scenario could be then used to populate couple of the notice policies. My understanding is that it's better to rely on actual factories and not hard coded values.  I think we have other hard coded responses here. I think we should probably also clean those perhaps in another PR.

Reply:
I think the factory for notice-policy will be added in the scope of another story, which is already in the upcoming sprint.